node/src/accountant/payment_adjuster/logging_and_diagnostics/log_functions.rs

```diff
@@ -160,12 +181,12 @@ pub fn log_insufficient_transaction_fee_balance(
 ) {
```

Comment:
I can't add a comment at a higher level, the suggestion here is as you mentioned that following renames should be performed:

`cw_required_transactions_count` -> `requested_tx_count`
`limiting_count` -> `feasible_tx_count`

Also, these arguments should be together.